clients/react-adapter.rst

```diff
@@ -1,2 +1,78 @@
 React Adapter
 =============
+
+An HTTPlug adapter for the `React Http client`_.
+
+Installation
+------------
+
+To install the React adapter, which will also install React itself (if it was
+not yet included in your project), run:
+
+.. code-block:: bash
+
+    $ composer require php-http/react-adapter
+
+Usage
+-----
+
+The React client adapter needs a :ref:`message factory <message-factory>` in
+order to to work::
+
+    use Http\Adapter\React\Client;
+
+    $client = new Client($messageFactory);
+
+For simplicity, all the required objects are instantiated automatically if not
+explicitly specified:
+
+:React\EventLoop\LoopInterface: The event loop used inside the React engine.
+:React\HttpClient\Client: The HTTP client instance that must be adapted.
+
+If you need more control on the React instances, you can inject them during
+initialization::
+
+    use Http\Adapter\React\Client;
+
+    $eventLoop = React\EventLoop\Factory::create();
+    $dnsResolverFactory = new React\Dns\Resolver\Factory();
+    $dnsResolver = $dnsResolverFactory->createCached('8.8.8.8', $loop);
+
+    $factory = new React\HttpClient\Factory();
+    $reactHttp = $factory->create($loop, $dnsResolver);
+
+    $adapter = new Client($messageFactory, $eventLoop, $reactHttp);
+
+If you choose to inject a custom React HTTP client, you must inject the loop
+used during its construction. But if you already use an EventLoop inside your
+code, you can inject only this object.
+
+For more simplicity, there is a ``ReactFactory`` which can be used to initialize
```

Comment:
I would remove "For more simplicity" redundant with the one above.

Maybe : You can also use a `ReactFactory` in order to initialize React instances::